### DISCUSSION TITLE
Expo and Native Modules

### DISCUSSION THREAD

**Discussion by RadouaneEzzaggab** [obytes/react-native-template-obytes#167] [Q&A] [answered]
Hi guys, can you ensure to us is that all native modules are fully compatible with expo
this is annoyed me while thinking to use expo

**Comment by yjose** [comment, marked as answer]
Yes, we are using the Expo Dev Client with the template, which means you can install any native dependencies that work with React Native CLI. But if the native dependency requires updating some native file, you need to create a simple plugin for it.

During the last year working with the Expo Dev Client, we didn't face any issues with native libraries.

**Comment by RadouaneEzzaggab** [comment]
Nice, Thank you so much.
It's been a few years i am comfused to use expo. But from now i will try it at least with simple projects.
